src/app/my-movies/my-movies.component.html

```diff
@@ -0,0 +1,4 @@
+<label for="sort">Sortuj:</label>
+<select [(ngModel)]="message.sort" id="sort" name="sort">
+  <option *ngFor="let sort of sorts" [value]="sort">{{sort}}</option>
+</select>
```

Comment:
roche to jest bez sensu że w my-movies nie ma filmów tylko samo sortowanie :p ale może to jeszcze nie skończona robota?

Reply:
Dokładnie wczorajsza nie skończona robota. W tym komponencie znajdą się jeszcze inne funkcje.